standalone-metastore/metastore-rest-catalog/src/main/java/org/apache/iceberg/rest/HMSCachingCatalog.java

```diff
@@ -63,6 +69,96 @@ public List<Namespace> listNamespaces(Namespace nmspc) throws NoSuchNamespaceExc
     return hiveCatalog.listNamespaces(nmspc);
   }
 
+  protected void cacheInvalidateInc(TableIdentifier tid) {
+    // This method is intentionally left empty. It can be overridden in subclasses if needed.
```

Comment:
what subclasses, please do not over-engineer

Reply:
The test-harnessed version of the cache uses those callback to check the feature.